### ISSUE TITLE
Feature request - Link to the documentation

### ISSUE THREAD

**Issue by luiscontrerasdo** [Icinga/icingaweb2#4269]
## Is your feature request related to a problem? Please describe.
When you install a module and it is displayed in the module section in IcingaWeb2, it has some information related to it, but there is no example of use or a link that point to that information.

## Describe the solution you'd like
It could be help full to have a link pointing to the module information or tutorial as well.

![image](https://user-images.githubusercontent.com/6249657/98534423-d4f29200-225a-11eb-91a3-5f041160a000.png)

**Comment by Al2Klimov** [comment]
@nilmerg Do you consider an additional module.info field a valid solution?

**Comment by nilmerg** [comment]
No. I didn't thought about that in detail yet and don't want to now.
